src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -27,6 +27,55 @@ injections:
         end: '(\})'
         endCaptures:
           1: { name: 'punctuation.curlybrace.close.cs' }
+  # 'L:meta.attribute':
+  #   patterns:
+  #     - name: 'meta.attribute.razor.explicit'
+  #       begin: '(")(@)\('
+  #       beginCaptures:
+  #         0: { name: 'keyword.control.cshtml' }
+  #         1: { patterns: [ include: '#transition' ] }
+  #       # begin: '"(@)\('
+  #       # beginCaptures:
+  #       #   0: { name: 'keyword.control.cshtml' }
+  #       #   1: { patterns: [ include: '#transition' ] }
+  #       patterns:
+  #         - include: 'source.cs#expression'
+  #       end: '\)'
+  #       endCaptures:
+  #         0: { name: 'keyword.control.cshtml' }
+  #     - name: 'meta.attribute.razor.implicit'
+  #       begin: '("[^@]*)(@)'
+  #       beginCaptures:
+  #         0: { name: 'keyword.control.cshtml' }
+  #         1: { patterns: [ include: '#transition' ] }
+  #       patterns:
+  #         - include: '#await-prefix'
+  #         - include: '#implicit-expression-body'
+  #       end: '(?=[\s<>\{\}\)\]])(?=([^"]*")'
+
+
+  'L:meta.attribute string.quoted.double.html':
+    patterns:
+      - name: 'meta.attribute.razor.explicit'
+        begin: '@\('
+        beginCaptures:
+          0: { name: 'keyword.control.cshtml' }
+          1: { patterns: [ include: '#transition' ] }
```

Comment:
So the 0 capture group represents everything captured and you don't technically have a `1` capture group. To make that the case you'd need to update the `begin` to be `(@)\(`. Basically having wrapping the @ symbol in parens captures it..

Reply:
Thanks for the explanation, I appreciate it! 

Updating `begin` to be `(@)\(` crashes the whole grammar, however, it seems the `0: { name: 'keyword.control.cshtml' }` makes `{ patterns: [ include: '#transition' ] }` redundant, so I've just removed it altogether.